_includes/newsletter.html

```diff
@@ -0,0 +1,23 @@
+<div class="guides-section-white">
+    <div class="row text-center">
+        <div class="col-md-6 col-md-offset-3 newsletter" >
+            <img src="/assets/img/guides/img-newsletter.png" alt="newsletter_image" class="img-center"><br>
+            <label>Newsletter</label>
+            <p>Subscribe to the Gruntwork Newsletter to get notified when updates and new guides are released. No spam. Ever.</p>
+            <form id="newsletter-form" class="col-xs-10 col-xs-offset-1" method="post" >
```

Comment:
How does this work? Shouldn't it POST to a MailChimp URL like http://gruntwork.io/newsletter?

Reply:
This is outdated. I hadn't pushed it